diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+ ~grace - wdi - try
```

Comment:
The current working directory is /Users/grace/wdi/try.  We see that running pwd yields "/Users/grace/wdi".  We ran "mkdir try" immediately afterward to create the "try" directory and then we entered it with "cd try".